packages/x-charts-pro/src/internals/plugins/useChartProExport/exportImage.ts

```diff
@@ -0,0 +1,92 @@
+import ownerDocument from '@mui/utils/ownerDocument';
+import { createExportIframe, loadStyleSheets } from './common';
+import { ChartImageExportOptions } from './useChartProExport.types';
+
+export const getDrawDocument = async () => {
+  let module;
+  try {
+    module = await import('rasterizehtml');
+  } catch (error) {
+    throw new Error(
+      `MUI X: Failed to import 'rasterizehtml' module. This dependency is mandatory when exporting a chart as an image. Make sure you have it installed as a dependency.`,
+      { cause: error },
+    );
+  }
+
+  return module.drawDocument;
+};
+
+export async function exportImage(
+  element: HTMLElement | SVGElement,
+  { fileName, type, quality }: ChartImageExportOptions = { type: 'image/png', quality: 0.9 },
```

Comment:
Decided to default quality to 0.9 to ensure default quality isn't browser-dependent.